lib/lab_manager/provider/vsphere.rb

```diff
@@ -94,7 +103,7 @@ def create_vm(opts = {})
 
       VSphere.with_connection do |vs|
         dest_folder = opts[:dest_folder]
-        vm_name = opts[:name] || 'lm_' + SecureRandom.hex(8)
+        vm_name = opts[:name] || "#{compute.image}-#{SecureRandom.hex(8)}"
```

Comment:
Just idea, not mandatory - this guid appending is respecting the old convention, but when you think of it, it should just guarantee uniqueness. What about adding something unique && informative?

So instead of TEMPLATE_XPx86-ab12cd34, we could have TEMPLATE_XPx86-20160721-085912300

...so, the appended information would be: YYYYMMDD-hhmmssxxx
(where xxx is millisecond)

This would add information about the age of machine, useful for maintenance tasks in virtual environment.
I do not insist on this format, but something like that - unique with informative value.